stress/basic.go

```diff
@@ -245,7 +245,26 @@ type BasicClient struct {
 	Concurrency   int      `toml:"concurrency"`
 	SSL           bool     `toml:"ssl"`
 	Format        string   `toml:"format"`
-	addrId        int
+
+	addrId   int
+	r        chan<- response
+	interval time.Duration
+}
+
+func (c *BasicClient) retry(b []byte, backoff time.Duration) {
+	bo := backoff + time.Duration(500*time.Millisecond)
```

Comment:
To increase the backoff for each additional retry.  An exponential backoff could look like:
```
// Function is invoked with the arg time.Duration(1)
if backoff != time.Duration(1) {
  bo := backoff * time.Duration(2*time.Millisecond)
} else {
  bo := time.Duration(500*time.Millisecond)
}
```